dotnet/samples/KernelSyntaxExamples/Getting_Started/Step1_Create_Kernel.cs

```diff
@@ -0,0 +1,38 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using System.Threading.Tasks;
+using Microsoft.SemanticKernel;
+
+/**
+ * This example shows how to create and use a <see cref="Kernel"/>.
+ */
+// ReSharper disable once InconsistentNaming
```

Comment:
Rather than put resharper front and center in all these examples, can we somehow centrally disable these rules so we don't need to continually reference it in source?

Reply:
Roger is going to handle this

Reply:
Thanks. I don't use resharper so it's just extra noise to me :)